enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/security/KeyValidator.java

```diff
@@ -127,31 +106,18 @@ public boolean validateScopes(TokenValidationContext validationContext) throws E
         return scopesValidated;
     }
 
-    private boolean validateSubscriptionDetails(String uuid, String context, String version, String consumerKey,
-            String keyManager, APIKeyValidationInfoDTO infoDTO) throws EnforcerException {
-        boolean defaultVersionInvoked = false;
-        String apiTenantDomain = FilterUtils.getTenantDomainFromRequestURL(context);
-        if (apiTenantDomain == null) {
-            apiTenantDomain = APIConstants.SUPER_TENANT_DOMAIN_NAME;
+    public static APIKeyValidationInfoDTO validateSubscription(String uuid, String apiContext, String apiVersion,
+                                                         String consumerKey, String keyManager) {
+        if (log.isDebugEnabled()) {
```

Comment:
can you remove the isDebugEnabled check and log in templated logging mode?

Reply:
When testing after making the changes I came across this https://github.com/wso2/product-microgateway/issues/2437. Tried adding templates to all the debug logs but it did not stop. Therefore removed some of the debug logs via this https://github.com/wso2/product-microgateway/pull/2432/commits/21dd29c99c97e3e87c32e09f0a24f4b51be5fbf8 which fixed the issue. The remaining changes are here. https://github.com/wso2/product-microgateway/pull/2432/commits/4ff61c3dac7e00bd0d2d4ad2ad32e6fee5daca6c